# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mono/gcc34-powerpc.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual conclifts catched by gcc >= 3.4.x.

  - Rene Rebe <rene@exactcode.de>

--- mono-1.1.3/mono/mini/exceptions-ppc.c.vanilla	2004-12-07 23:33:57.000000000 +0100
+++ mono-1.1.3/mono/mini/exceptions-ppc.c	2005-02-04 10:49:59.579882290 +0100
@@ -226,6 +226,9 @@
 	return start;
 }
 
+static gpointer
+mono_arch_get_throw_exception_generic (guint8 *start, int size, int by_name, gboolean rethrow);
+
 /*
  * arch_get_call_filter:
  *
